asabiris/formatter/jinja/service.py

```diff
@@ -28,8 +30,15 @@ def __init__(self, app, service_name="JinjaService"):
 			self.Variables = {}
 
 		self.Environment = jinja2.Environment()
+		# Inject 'now' function into the Jinja2 template's global namespace
+		self.Environment.globals['now'] = self._jinja_now
 		self._load_variables_from_json()
 
+	def _jinja_now(self):
+		timezone = pytz.timezone('Europe/Prague')
```

Comment:
Timezone must be taken from the tenant configuration, cannot be hardcoded.

Reply:
@ateska Okay, how do I extract timezone from tenant configuration?

Reply:
I guess that you know the tenant (if you don't then fallback to UTC timezone).
Then you need to query SeaCat Auth API ... using discover service.